### DISCUSSION TITLE
v2.12.1

### DISCUSSION THREAD

**Discussion by jgraichen** [jgraichen/redmine_dashboard#227] [Announcements]
v2.12.1 contains a fix for a Zeitwerk loader issue on Redmine 5.0.

### Fixes
* Zeitwerk autoloader issues on Redmine 5.0

##  Looking for testers!

Unfortunately, I do not have access to any larger Redmine project anymore, nor am I using Redmine anywhere anymore. I can only test with some artificially constructed local project that cannot represent any real project or usage. I would really appreciate if you can test v2.12.0 in your project or organization and report any findings (or their absence). Thank you!

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraichen/redmine_dashboard/releases/tag/v2.12.1'>v2.12.1</a>.</em>

**Comment by simonello** [comment]
Firstly , thank you for the fixes.

Started testing today on our Redmine UAT Server.   After a week will roll to production and let you know how we got on.
